templates/base/client/prototype-ui/App.tsx.hbs

```diff
@@ -130,6 +131,30 @@ function Login({ setToken }: { setToken: (token: string) => void }) {
               Login (Anonymous)
             </button>
           </div>
+{{else if (eq @key "nickname")}}
+          <div className="w-6/12 mb-4">
+            <form onSubmit={() =>
```

Comment:
to stay consistent with the rest of the prototype-ui we should use `button.onClick` instead of `form.onSubmit`

Reply:
fixed!